### PR TITLE
Finalize Rule

### PR DESCRIPTION
**Pull request by tnuanchuay**
- no round feature
- get unconfirmed ticket

### CODE REVIEW

**Comment by varokas** [comment]
@tspn. using Yarn is great idea, please change README.md to mentioned how to use it with yarn

File: index.js [hunk 1]
```diff
@@ -130,24 +131,43 @@ app.post('/cancel', function(req, res) {
...
+ * 
+ */
+app.get('/tickets/unconfirmed', (req, res) => {
+    const unconfirmed = book.getUnconfirmed()
+    res.json({
+        count: unconfirmed.length,
+        seats: unconfirmed
+    }).end()
+})
+
```
Comment (varokas) [inline index.js]: when are we going to call this endpoint?

Comment (varokas) [reply index.js]: I think what we meant in https://github.com/bigbearsio/strength/issues/6, was that `/bookings` will also return number of unconfirmed seats. So the client knows when to stop trying. 

We do not need to have another endpoint to get unconfirmed seats.

Comment (tspn) [reply index.js]: I'll remove this endpoint +
we already have endpoint /remaining that return available + reserve expired. this should help to get available seat as well.

Comment (tspn) [reply index.js]: now, I so confuse we have /booking and /remaining, maybe I created it but forget it

Comment (tnuanchuay) [reply index.js]: /remaining could help us to get number of available. it isn't return number, but it return seat that available.

(/remaining return available seats and also reserve expired seat)

**Comment by varokas** [comment]
I still think you haven't address adding unconfirmed seats number to `/remaining`. but let's merge this and put that in another issue

**Pull request merged**